VoodooI2CHID/VoodooI2CTouchscreenHIDEventDriver.cpp

```diff
@@ -190,20 +184,20 @@ void VoodooI2CTouchscreenHIDEventDriver::fingerLift() {
     uint64_t now_abs;
     clock_get_uptime(&now_abs);
 
-    // This seems strange, but we go to hover mode after right clicking, and we need to execute a click if we've moved
-    if (right_click && moved_during_right_click) {
+    if (!is_dragging && (!right_click || moved_during_right_click)) {
         dispatchDigitizerEventWithTiltOrientation(now_abs, last_id, kDigitiserTransducerFinger, 0x1, LEFT_CLICK, last_x, last_y);
```

Comment:
You'll notice that now clicks only happen inside of this timed function. Also, I only ever send `LEFT_CLICK` here, whereas it used to send `transducer->tip_switch.value()`. Do we think that might be an issue?